views/allbooks.handlebars

```diff
@@ -1,25 +1,27 @@
 <body>
 <h1>These are all books that are available on BookSwap</h1>
 
-<table>
-    <thead>
-        <th>id</th>
-        <th>title</th>
-        <th>author</th>
-        <th>genre</th>
-        <th>isbn</th>
-        <th>imgUrl</th>
-    </thead>
-    <tbody>
+
+<div class="shelf">
```

Comment:
Trying to change the column count using @include media-breakpoint-only.

I get this error when I enter this into the css file:
Unknown at rule @includecss(unknownAtRules).

After a search I learned that  @include is a feature of sass. Does this mean it would only be recognized in scss file? I don't want to add anything we didn't agree on using.

Reply:
https://stackoverflow.com/a/43117538 After some more searching i found this suggestion to achieve the same thing using css only

Reply:
they also mention the same thing in the following link https://getbootstrap.com/docs/4.1/layout/overview/
in the responsive breakpoints section

Reply:
Please see comment above where I note: "We also only want to display the title/author/point cost on this page."

This has not been resolved and typically you do not resolve your own conversations if a reviewer has requested changes. I will approve with this solution after you make it display the correct fields and then I will look at refactoring it as I don't believe this is the ideal solution and won't work well with additional functionality in the upcoming sprints.